tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -15,13 +15,8 @@ if ! command -v git ; then
         exit 1
     fi
 fi
-# 0.17.0 is the oldest version of letsencrypt-auto that has precompiled
-# cryptography and the tagged commit is in master. 0.16.0 was the first version
-# to use precompiled cryptography, but the release PR was squashed losing the
-# commit. We want to use a precompiled version of cryptography for stability.
-# Previous versions that have to compile against OpenSSL on installation
-# started failing on newer distros with newer versions of OpenSSL.
-INITIAL_VERSION="0.17.0"
+# 0.18.0 is the oldest version of letsencrypt-auto that works on Fedora 26+.
```

Comment:
Fedora 26 is EOL, and so 27 in fact (end of last year). Why we do not set the minimum version here in regards to the last non EOL, Fedora 28?

Reply:
0.18.0 is also needed for Fedora 28 which is the Fedora version being used in this PR.

I said Fedora 26+ because that marks the version where Fedora changed package names requiring a change to certbot-auto which is documented at https://github.com/certbot/certbot/blob/master/CHANGELOG.md#fixed-26.

Reply:
OK !